tests/test_hdf5.py

```diff
@@ -232,7 +232,7 @@ def test_out_of_memory_unsorted_indices(self):
         assert_raises(TypeError, dataset.get_data, handle, [7, 4, 6, 2, 5])
         dataset.close(handle)
 
-    def test_in_of_memory_example_scheme(self):
+    def test_in_memory_example_scheme(self):
```

Comment:
https://github.com/mila-udem/fuel/pull/289/files#diff-7c09a0dc93ce18f66103684bcb4d2b01R244 tests in-memory dataset.